packages/test/package.json

```diff
@@ -41,13 +47,15 @@
     "@solarwinds-apm/module": "workspace:^",
     "@types/chai": "^4.3.6",
     "@types/chai-as-promised": "^7.1.6",
-    "@types/node": ">=16.18 <17 || >=18.8 <19 || >=20",
+    "@types/mocha": "10.0.6",
+    "@types/node": "^18.19.0 || >=20.6.0",
     "c8": "^9.1.0",
     "chai": "^4.3.10",
     "chai-as-promised": "^7.1.1",
     "globby": "^11.1.0",
+    "mocha": "^10.3.0",
     "semver": "^7.5.4",
-    "tsx": "^4.1.3"
+    "ts-node": "^11.0.0-beta.1"
```

Comment:
Both `tsx` and the native Node.js test runner were having issues with code coverage on Node 20 so I replaced them with `ts-node` and `mocha`. Hopefully this can now stay the way it is.